src/dev/code_coverage/ingest_coverage/README.md

```diff
@@ -0,0 +1,123 @@
+
+# Convert Code Coverage Json Summary and Send to ES
+
+## Setup
+Run converage using json-summary, put the index, create the index pattern,
+and run the converter to post docs to the index.  
+```
+PUT kibana_coverage
```

Comment:
I think this index is named `kibana_code_coverage` not `kibana_coverage`?

I'm wondering if we should just reference the mapping in src/dev/code_coverage/ingest_coverage/index_mapping.md instead of duplicating it here?  Otherwise we have to keep them in sync.

Also, isn't there also a mapping for the kibana_total_code_coverage ?